.ci/test-local-tap.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -eo pipefail
+
+. "${0%/*}/libfunctions.sh"
+
+echo '--- :evergreen_tree: Configuring Environment'
+exec 9>&1 # enable tee to write to STDOUT as a file
+test_bottle
+
+echo '+++ :beer: Homebrew Tap and Install'
+perform 'brew update'
+if [[ ! $(python3 --version 2>/dev/null) ]]; then
+   perform 'brew install python3'
+fi
+perform "sed -i.bk -e 's/url \".*\"/url \"http:\/\/127.0.0.1:7800\"/' ${BUILDKITE_PIPELINE_SLUG}/*.rb"
+perform "pushd ${BUILDKITE_PIPELINE_SLUG} && git add *.rb && git commit -m 'test it!' && popd"
+perform "brew tap eosio/eosio ${BUILDKITE_PIPELINE_SLUG}"
+perform '{ python3 -m http.server 7800 & } && export HTTP_SERVER_PID=$!'
+perform 'sleep 20s'
+perform 'brew install eosio'
+perform 'kill $HTTP_SERVER_PID'
```

Comment:
Should this have an "or no-op" (`|| :`) in case this `kill` command fails? If we leave an HTTP server running, that shouldn't be a test failure....right?
```bash
perform 'kill $HTTP_SERVER_PID || :'
```

Reply:
I don't see that failing, but if it does, it will hang the job causing the timeout to kill it because buildkite will see the server open and it will still have the parent PID associated with it.

Reply:
That said, there is a possibility that `brew install eosio` fails (an actual error code), and the job will fail due to timeout because of `set -e`.